spec/support/media_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.shared_examples "a media controller" do
+
+	
+  describe "GET 'index'" do
+
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET 'new'" do
+  	it "renders the 'new' view" do
+  		get :new
+  		expect(subject).to render_template("new")
+  	end
+  end
+
+  describe "GET 'show'" do
+  	it "renders the show view" do
+  		get :show, id: media.id
+  		expect(response.status).to eq 200
```

Comment:
How can you try to make this test different than the GET 'index' method? Could you test to see if the media is the right one you're supposed to be showing?